processing/src/main/java/io/druid/segment/incremental/IncrementalIndexAdapter.java

```diff
@@ -269,32 +266,15 @@ public ColumnCapabilities getCapabilities(String column)
   }
 
   @Override
-  public BitmapIndexSeeker getBitmapIndexSeeker(String dimension)
+  public IndexedInts getBitmapIndex(String dimension, int dictId)
   {
-    final Map<String, MutableBitmap> dimInverted = invertedIndexes.get(dimension);
-    if (dimInverted == null) {
-      return new EmptyBitmapIndexSeeker();
+    if (dictId >= 0) {
+      final Indexed<String> dimValues = getDimValueLookup(dimension);
+      String value = Strings.nullToEmpty(dimValues.get(dictId));
```

Comment:
Yes, you are right, NullValueConverterDimDim will convert empty to null, we need convert it back to the actual values, because getBitmapIndex relies on the actual values stored in DimDim.